lib/nodejs/lib/thrift/connection.js

```diff
@@ -182,10 +170,18 @@ Connection.prototype.initialize_retry_vars = function () {
   this.retry_timer = null;
   this.retry_totaltime = 0;
   this.retry_delay = 150;
-  this.retry_backoff = 1.7;
+  this.retry_backoff = 1.5;
   this.attempts = 0;
 };
 
+Connection.prototype.flushQueue = function () {
+  var current_queue = this.offline_queue.slice();
```

Comment:
I don't think we need a copy here since we're throwing away the old one right away.